### PR TITLE
Star wars landing page css updates

### PR DESCRIPTION
**Pull request by caleybrock**
desktop
screen shot 2015-11-09 at 3 10 02 pm
tablet
screen shot 2015-11-09 at 3 10 10 pm
mobile
screen shot 2015-11-09 at 3 10 22 pm

### CODE REVIEW

**Comment by caleybrock** [comment]
This also makes sure the tutorial doesn't open in a new tab.

File: pegasus/sites.v3/code.org/public/css/starwars.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.starwars-container {
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/starwars.css:1]: this file is back :)

File: pegasus/sites.v3/code.org/public/css/starwars.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+@media screen and (min-width: 0px) and (max-width: 970px) {
+  .starwars-tutorial {
+    width: 50%;
+    float: left;
+    height: 220px;
+	  padding: 5px;
+    padding-top: 10px;
+    position: static;
+	}
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/css/starwars.css]: weird indenting?

Comment (breville) [inline pegasus/sites.v3/code.org/public/css/starwars.css]: looks like some tabs too

**Comment by breville** [comment]
nice work!

File: pegasus/sites.v3/code.org/public/css/starwars.css (hunk 2 above)
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/css/starwars.css]: it's weird that sublime didn't show this. updated.

**Comment by caleybrock** [comment]
@breville good to merge?

**Pull request merged**